packages/remix-dev/cli/create.ts

```diff
@@ -533,6 +641,23 @@ export async function validateTemplate(input: string) {
 
       switch (response.status) {
```

Comment:
Also a potential DX improvement, if I run my command without providing the GITHUB_TOKEN env variable, I get this error:

```🚨 The template could not be verified. The server returned a response with a 401 status. Please double check that the template is a valid GitHub repository and try again.```

We could add a `401` case here explaining that there's an Authorization problem, and a valid GitHub token with access to this repo must be provided.

Reply:
for sure, done.